src/shared/lib/StoreUtils.ts

```diff
@@ -1830,8 +1830,9 @@ export function filterAndAnnotateMolecularData(
             getPutativeDriverInfo(datum),
             discreteCnaProfileIds
         );
-        annotatedDatum.hugoGeneSymbol =
-            entrezGeneIdToGene[datum.entrezGeneId].hugoGeneSymbol;
+        annotatedDatum.hugoGeneSymbol = (
+            entrezGeneIdToGene[datum.entrezGeneId] || datum.gene
+        ).hugoGeneSymbol;
```

Comment:
This is a workaround for the entrez gene ids which we don't have a corresponding gene in the `entrezGeneIdToGene` map. Not sure if we really need `entrezGeneIdToGene`, it seems to me that we can always use `datum.gene`.